actiontext/test/dummy/config/environments/test.rb

```diff
@@ -23,7 +23,7 @@
   config.action_controller.perform_caching = false
 
   # Raise exceptions instead of rendering exception templates.
-  config.action_dispatch.show_exceptions = false
+  config.action_dispatch.show_exceptions = :none
```

Comment:
I believe the dummy configs should match what's generated in new applications (`:rescuable`)